File: 백준/Bronze/2562. 최댓값/최댓값.js
const fs = require("fs");
const path = require("path");
const input = fs
  .readFileSync(
    process.platform === "linux"
      ? "/dev/stdin"
      : path.join(__dirname, "/input.txt"),
  )
  .toString()
  .trim()
  .split("\n");

let max = 0;
let idx = 0;

for (let i = 0; i < 9; i++) {
  const n = Number(input[i]);
  max = Math.max(max, n);
  if (max === n) idx = i + 1;
}
console.log(max);
console.log(idx);

File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <iostream>

using namespace std;

int main()
{
    int max=0;
    int idx=0;
    for(int i=1;i<=9;i++){
        int num;
        cin >> num;
        if(num>max){
            max=num;
            idx=i;
        }
    }
    cout <<max << "\n" << idx << endl;
}